javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2586,6 +2586,25 @@ public static boolean isVarArgs(MethodInvocationTree invok) {
     return isVarArgs(elementFromUse(invok), invok.getArguments());
   }
 
+  /**
+   * Returns true if the given method invocation is a zero-argument varargs invocation.
+   *
+   * <p>This method checks two things: the first is that the given method invocation is a varargs
+   * invocation. The second is that the size of the argument list is equal to the size of the number
+   * of parameters minus one.
+   *
+   * <p>The second check handles the case where an empty varargs argument is passed for methods that
+   * <i>only</i> have the single varargs parameter, <i>and</i> methods that could have additional,
+   * non-varargs parameters.
+   *
+   * @param invok the method invocation
+   * @return true if the given method invocation is a zero-argument varargs invocation
+   */
+  public static boolean isZeroArgumentVarArgs(MethodInvocationTree invok) {
+    int numParams = elementFromUse(invok).getParameters().size();
+    return TreeUtils.isVarArgs(invok) && (invok.getArguments().size() == numParams - 1);
```

Comment:
@mernstdaikon what is your preference here on extracting the second operand of the `&&` to `numParams`? I prefer it for readability (since it's a longer chain of methods) but I could easily inline it, if you prefer.

Reply:
One possibility would be:
```
if (!TreeUtils.isVarArgs(invok)) { return false; }
int numParams = elementFromUse(invok).getParameters().size();
return (invok.getArguments().size() == numParams - 1);
```
Inlining also works.